tests/testthat/test-assertions.R

```diff
@@ -687,7 +687,6 @@ test_that("assert_numeric_vector Test 39: no error if `arg` is NULL and optional
   )
 })
 
-# assert_integer_scalar ----
 ## Test 40: error if `arg` is not an integer scalar ----
 test_that("assert_integer_scalar Test 40: error if `arg` is not an integer scalar", {
```

Comment:
This seems inccorrect - the integer and numeric stuff

Reply:
ah, yes, the text needed to updated too
![image](https://github.com/pharmaverse/admiraldev/assets/26774684/4e16ef79-6132-47ea-981f-38333735692b)